src/storage/inmemory.storage.ts

```diff
@@ -57,7 +57,7 @@ export class InMemoryStorage implements IStorer {
     }
   }
 
-  private entryIsStillValid(entry: DataEntry<unknown>): boolean {
-    return entry.expiresAt > Date.now() || entry.expiresAt === Infinity;
```

Comment:
If `entry.expiresAt === Infinity` the first condition will always return true because `Infinity > Date.now()` for whatever value `Date.now()` wants to be. @lilpolymath

Reply:
I don't understand what you mean.

`entry.expiresAt` will always have to be set to Infinity by the user, we'll never default to Infinity, that's why I removed
![CleanShot 2024-06-24 at 10 42 40@2x](https://github.com/BolajiOlajide/now-playing/assets/25608335/35ad00d3-4673-4ccc-b9e2-f986fdcae9e5)

The `duration` field is now required and will always be set so the only way we'd get `Infinity` is if the user themselves set it to `Infinity`